### ISSUE TITLE
Inconsistent results

### ISSUE THREAD

**Issue by Joselinejamy** [TeamHG-Memex/sklearn-crfsuite#38]
On running the following code, am getting different recall, precision and f1 scores on different runs over the same dataset.

```
crf = sklearn_crfsuite.CRF(
    algorithm='lbfgs',
    c1=0.1,
    c2=0.1,
    max_iterations=100,
    all_possible_transitions=True
)
crf.fit(X_train, y_train)

```

Came to know that this is expected by going through one of the issues, but is it possible to set any random seed to a constant value to get consistent results for same input ?

**Comment by severinsimmler** [comment]
+1

**Comment by severinsimmler** [comment]
This is related to https://github.com/TeamHG-Memex/sklearn-crfsuite/issues/9.

**Comment by franklevasseur** [comment]
@severinsimmler, I actually think that it's not realted to #9 as both reported issues use different optimizers. The training with lbfgs optimizer does not call the `dataset_shuffle` function.

**Comment by liaeh** [comment]
I'm also having this problem... makes it very difficult to compare e.g. the benefit of including/excluding different features
